.github/workflows/ci_frontend.yml

```diff
@@ -0,0 +1,39 @@
+name: 'CI - Frontend'
+
+on:
+  push:
+    branches:
+      - master
+      - dev
+      - 'releases/**'
+    paths-ignore:
+      - 'RetrospectiveExtension.Backend/**'
+  pull_request:
+    branches:
+      - master
+      - dev
+      - 'releases/**'
+    paths-ignore:
+      - 'RetrospectiveExtension.Backend/**'
+  workflow_dispatch:
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    name: Build and Test
+
+    steps:
+    - uses: actions/checkout@v2
+      name: Check out repository
+
+    - run: |
```

Comment:
I am curious which version of node and npm does this use by default. It could be differernt than our current dev/prod setup

Reply:
This is a valid concern and I think GitHub action allows us to install a specific verision of npm like this:

```yaml
      - name: 'change version'
        uses: reedyuk/npm-version@1.1.1
        with:
          version: '1.0.1'
          package: 'subproject-directory/'
          git-tag-version: 'true'
```

I've created task [#1499](https://dev.azure.com/csebostoncrew/Retrospectives-AzDO-Extension/_workitems/edit/1499) so that we can address it in the next PR after discussing it broadly with our team.

Reply:
great! thx Phong